lms/djangoapps/courseware/views/views.py

```diff
@@ -341,6 +341,17 @@ def get_last_accessed_courseware(course, request, user):
                 return url
         return None
 
+
```

Comment:
Let's recheck this piece of code

Reply:
done, it was a mistake in copy and pasting of the code.